db/controllers.js

```diff
@@ -7,3 +7,12 @@ exports.getCategories = (request, response, next) => {
     })
     .catch(next);
 };
+
+exports.getReviews = (request, response, next) => {
+  fetchAllReviews()
+    .then((reviews) => {
+      response.status(200).send(reviews);
```

Comment:
Good stuff, but the returned value should be an object with a key of reviews, and a value of the reviews array 🔍